desec/resource_rrset.go

```diff
@@ -44,7 +44,7 @@ func resourceRRSet() *schema.Resource {
 				Type:         schema.TypeString,
 				Required:     true,
 				ForceNew:     true,
```

Comment:
Needs to be re-aligned for gofmt if `ValidateFunc` is completely removed:
```suggestion
				Type:     schema.TypeString,
				Required: true,
				ForceNew: true,
```